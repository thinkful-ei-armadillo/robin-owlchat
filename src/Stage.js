import React from 'react';

export default function Stage (props){
    const session = props.onStage ? 'stage view' : 'stage hidden';
return (
    <div className={session}>
        <div className="contact-on-stage" data-tooltip={props.name}>
            <div className="contact-on-stage-video-container">
                <img src={props.avatar} alt={props.name} className="user-avatar contact-on-stage-avatar">
                </img>
                <div className="contact-on-stage-details">
                    <span className="contact-on-stage-details-text">
                    {props.name}
                    </span>
                    <button className="contact-on-stage-details-button"></button>
                </div>
            </div>
        </div>
    </div>
)
}