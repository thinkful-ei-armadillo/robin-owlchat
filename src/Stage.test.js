import React from 'react';
import ReactDOM from 'react-dom';
import Stage from './Stage';
import renderer from 'react-test-renderer';


it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Stage />, div);
    ReactDOM.unmountComponentAtNode(div);
  });
  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<Stage 
        name = 'sdfff'
        avatar = "https"
        onStage = {false}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
    });