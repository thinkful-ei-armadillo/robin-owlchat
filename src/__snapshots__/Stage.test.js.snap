// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div
  className="stage hidden"
>
  <div
    className="contact-on-stage"
    data-tooltip="sdfff"
  >
    <div
      className="contact-on-stage-video-container"
    >
      <img
        alt="sdfff"
        className="user-avatar contact-on-stage-avatar"
        src="https"
      />
      <div
        className="contact-on-stage-details"
      >
        <span
          className="contact-on-stage-details-text"
        >
          sdfff
        </span>
        <button
          className="contact-on-stage-details-button"
        />
      </div>
    </div>
  </div>
</div>
`;
